psycop/common/model_training_v2/hyperparameter_suggester/test_hyperparam_search.cfg

```diff
@@ -1,4 +1,12 @@
 [model]
+@suggesters = "suggester_space"
+
+[model.*.logistic_regression]
+@estimator_steps = "logistic_regression_suggester"
+C = {"low": 0.1, "high": 1.0, "logarithmic": "True"}
+l1_ratio = {"low": 0.01, "high": 1.0, "logarithmic": "True"}
+
+[model.*.logistic_regression_2]
```

Comment:
This is currently unused. Add it to the test to see if we different results are obtained?